### ISSUE TITLE
Request: uchime score in fasta label

### ISSUE THREAD

**Issue by colinbrislawn** [torognes/vsearch#155]
I want to second a [suggestion](https://github.com/torognes/swarm/issues/43#issuecomment-76920764) that @frederic-mahe made awhile ago. 

I would find it convenient to have the uchime score appended to the fasta read label, like this:
```
>derep_8652;size=26;uchimedenovo=0.2594;
GACGGAGGAGGCAA
```

Ideally, the label would be different for `uchime_denovo` and `uchime_ref` so you could run a file through both methods and get annotations like this:
```
>derep_8652;size=26;uchimedenovo=0.2594;uchimeref=1.3254;
```

Thanks! This software is great!

**Comment by torognes** [comment]
Thanks for the suggestion, @colinbrislawn ! I'll add this in the next release.

**Comment by torognes** [comment]
This issue should be solved in version 1.10.0 just released. Use the new `--fasta_score` option.

**Comment by colinbrislawn** [comment]
:star2: Oh this is awesome! I look forward to using this great feature.
